hyppo/conditional/kci.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from scipy.stats import gamma
+from sklearn.gaussian_process import GaussianProcessClassifier
+from sklearn.gaussian_process.kernels import RBF
+
+from .base import ConditionalIndependenceTest, ConditionalIndependenceTestOutput
+
+
+class KCI(ConditionalIndependenceTest):
+    r"""
+    Kernel Conditional Independence Test Statistic and P-Value.
+    This is a conditional indpendence test utilizing a radial basis 
+    function to calculate the kernels of two datasets. The trace
+    of the normalized matrix product is then calculated to extract the test 
+    statistic. A Gaussian distribution is then utilized to calculate
+    the p-value given the statistic and approximate mean and variance
+    of the trace values of the independent kernel matrices.
+    This test is consistent against similar tests..
+    """
+
+    def __init__(self, **kwargs):
+
+        ConditionalIndependenceTest.__init__(self, **kwargs)
+
+    def compute_kern(self, x, y):
```

Comment:
This is not really necessary, since compute_kern does the same thing. I would use that instead. If you have a specific theta as a default, pass that to the gamma function